internal/config/loader_git.go

```diff
@@ -21,10 +21,11 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// TODO: Should we normalise names for constants? lowercase vs uppercase
```

Comment:
I think we can maybe split them into 2 different const blocks but the Uppercased ones we want to keep as uppers as we can reference them from the outside e.g. tests and other implementations.

the lower case we don't want to export 😄